packages/create-fuels/src/cli.ts

```diff
@@ -139,18 +139,31 @@ export const runScaffoldCli = async ({
 
   fileCopySpinner.succeed('Copied template files!');
 
+  // Remove typegen files from gitignore
+  const gitignorePath = join(projectPath, '.gitignore');
+  const gitignoreContents = readFileSync(gitignorePath, 'utf-8');
+  const newGitIgnoreContents = gitignoreContents
+    .replace(/src\/sway-api\/contracts\n/g, '')
+    .replace(/src\/sway-api\/predicates\n/g, '')
+    .replace(/src\/sway-api\/scripts\n/g, '')
+    .replace(/src\/sway-api\/index.ts/g, '');
```

Comment:
Is there any reason you haven't kept the paths [as comments](https://github.com/FuelLabs/fuels-ts/pull/3175#discussion_r1761369499)?

Reply:
Forgot to include this change in the new approach. Added it back. Thanks for pointing it out

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3175/commits/d7147517acb035863184f5b68d9f10825338c2fe

Reply:
Resolving as fixed in above commit.